src/Compilers/Core/Portable/Compilation/Extensions.cs

```diff
@@ -61,6 +61,18 @@ public static TypeInfo GetTypeInfo(this SemanticModel semanticModel, SyntaxNode
         }
 
         /// <summary>
+        /// Gets natural type of expression, if any.
+        /// </summary>
+        /// <param name="semanticModel"></param>
+        /// <param name="expression">The syntax node to get information for.</param>
+        /// <param name="cancellationToken">A cancellation token that can be used to cancel the
+        /// process of obtaining the semantic info.</param>
+        public static ITypeSymbol GetNaturalType(this SemanticModel semanticModel, SyntaxNode expression, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            return semanticModel.GetNaturalType(expression, cancellationToken);
```

Comment:
Isn't this just calling itself?  Or is this calling some internal instnace member that it has access to?